manifests/init.pp

```diff
@@ -203,4 +203,16 @@
       mode    => $ctrlaltdel_override_mode,
     }
   }
+  if ($::osfamily == 'RedHat' and $::operatingsystemmajrelease == '6') {
```

Comment:
In the implementation for EL5, if require_single_user_mode_password_bool is false, then no configs are added. This seems to add configs either way and only changes the shell, which I don't understand.

Reply:
In RHEL6, the default file has the shell listed. The sushell is the no password shell which is the behavior you would see in EL5 if not touching the configs and the sulogin is the shell that prompts for the password.

Reply:
Thank you for the explanation and the code! Could you please implement this in the template as opposed to using file_line? The file_line resource is for files where you do not manage the entire thing or to build up files programmatically and should be avoided mixing with templates or sourced files.